### PR TITLE
Remove python2 code from deployment tools

### PR DESCRIPTION
**Pull request by Chitresh13**
Remove bunch of code written in python 2 from dist directory.

### CODE REVIEW

File: direct/src/dist/commands.py [hunk 1]
```diff
@@ -209,8 +173,7 @@ def get_data(path):
 del os
 """
 
-SITE_PY = SITE_PY3 if sys.version_info >= (3,) else SITE_PY2
-
+SITE_PY = SITE_PY3 if sys.version_info >= (3,)
```
Comment (rdb) [inline direct/src/dist/commands.py]: This is not actually valid Python syntax.  You don't need the `if` here.  In fact, you don't need the alias.  I suggest just removing this line and changing the original `SITE_PY3` to `SITE_PY`.

File: direct/src/dist/FreezeTool.py [hunk 2]
```diff
@@ -41,9 +41,6 @@
 if sys.version_info >= (3, 0):
```
Comment (Moguri) [inline direct/src/dist/FreezeTool.py]: We don't need to branch here anymore and can just set `startupModules` directly.

File: direct/src/dist/commands.py (hunk 1 above)
Comment (Chitresh13) [reply direct/src/dist/commands.py]: Sure I will sure do it.

Comment (Chitresh13) [reply direct/src/dist/commands.py]: Sure ! I will make those changes.

File: direct/src/dist/FreezeTool.py (hunk 2 above)
Comment (Chitresh13) [reply direct/src/dist/FreezeTool.py]: Ok ! I will set it

**Comment by rdb** [comment]
Superseded by #892

**Pull request closed without merging**